### ISSUE TITLE
PDO Error using PostgreSQL when visiting Dashboard

### ISSUE THREAD

**Issue by Timusan** [verbb/formie#222]
**Description**

When you visit the Formie forms overview dashboard (formie/forms) you expect to get a listing of forms, instead your get a "Server error occurred" message. Checking the logs, this is what happens:

`PDOException: SQLSTATE[42883]: Undefined function: 7 ERROR:  operator does not exist: integer ~~ unknown
LINE 7: ...elements_sites"."siteId"=$1) AND ("elements"."id" LIKE $2) A...
                                                             ^`
`HINT:  No operator matches the given name and argument types. You might need to add explicit type casts.`

This happens even with a fresh install of Formie on PostgreSQL 12.4. 

**Steps to reproduce**
1. Install Formie on a Craft installation running on PostgreSQL 12.4
2. Visit the forms overview dashboard

This also happens when visiting the Formie Submissions overview (formie/submissions)

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.2.28",
  "...": "..."
}
```

Then run `composer update`.

**Comment by Timusan** [comment]
Thanks for the fast reply. I have let Composer update to the dev branch and it clears *that* error, but brings a new one:

`Invalid text representation: 7 ERROR:  invalid input syntax for type integer: "none" in /vendor/yiisoft/yii2/db/Command.php:1293`

However, this error now only occurs when there are no forms made yet. Once you make a form you can now visit the dashboard(s) normally.

**Comment by engram-design** [comment]
Should also be fixed in the dev branch as well!

**Comment by Timusan** [comment]
Confirmed 👍

**Comment by engram-design** [comment]
Fixed in [1.3.0](https://github.com/verbb/formie/releases/tag/1.3.0)

**Comment by Timusan** [comment]
Hi Josh

This old issue seems to be back. Since the recent upgrade (Formie 1.6.21, Craft 3.7.63.1):

```
Undefined function: 7 ERROR:  operator does not exist: integer ~~ unknown
LINE 6: WHERE ("formie_submissions"."formId" LIKE $1) AND ("formie_s...
```

This is a PostgreSQL install. Could this be regression?
